src/gar.lua

```diff
@@ -1243,22 +1141,19 @@ end
 --- @return table result Result table containing updated gateway object and the delegates that were actually added
 function gar.allowDelegates(delegateAddresses, gatewayAddress)
 	local gateway = gar.getGateway(gatewayAddress)
-	if gateway == nil then
-		error("Gateway not found")
-	end
+	assert(gateway, "Gateway not found")
 
 	-- Only allow modification of the allow list when allowDelegatedStaking is set to false or a current allow list is in place
-	if gateway.settings.allowDelegatedStaking == true and not gateway.settings.allowedDelegatesLookup then
-		error("Allow listing only possible when allowDelegatedStaking is set to 'allowlist'")
-	end
+	assert(
+		gateway.settings.allowDelegatedStaking == true and not gateway.settings.allowedDelegatesLookup,
+		"Allow listing only possible when allowDelegatedStaking is set to 'allowlist'"
+	)
```

Comment:
need to fix

Reply:
https://github.com/ar-io/ar-io-network-process/pull/202/commits/f76447d3fd890ef960259e2552da5a3d5b0b3924